### DISCUSSION TITLE
机械革命翼龙 15Pro 迁移 Linux 遇到的一些问题 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#17] [Announcements]
# 机械革命翼龙 15Pro 迁移 Linux 遇到的一些问题 🐱 ネコのメモ帳

买了高性价比非大众电脑带来的后果，就是完全没有对 linux 等做过硬件上的适配（也不准备支持）。用了一周解决了大概三个问题：内置键盘失灵、蓝牙模块无效、显卡驱动装不上、休眠立马唤醒的问题，简单记录一下。

https://n.ova.moe/blog/%e6%9c%ba%e6%a2%b0%e9%9d%a9%e5%91%bd%e7%bf%bc%e9%be%99-15Pro-%e8%bf%81%e7%a7%bb-Linux-%e9%81%87%e5%88%b0%e7%9a%84%e4%b8%80%e4%ba%9b%e9%97%ae%e9%a2%98/

**Comment by qwq2519** [comment]
tql，我也买了这个，要不要退货

**Reply by MuelNova** [reply]
你可以试试 ubuntu 之类的，可能没这个问题

**Reply by MuelNova** [reply]
最好还是用 wsl

**Reply by qwq2519** [reply]
VM可以吗

**Reply by MuelNova** [reply]
vm 可以

**Reply by qwq2519** [reply]
谢谢

**Comment by leisiji** [comment]
我刚好也是遇到无法睡眠这个问题了，感谢分享
不过我的机器在解决 gpio 的问题后还是被唤醒了，后来用 pm_wakeup_irq 发现是 i8042，最后通过 `echo disabled > /sys/devices/platform/i8042/serio0/power/wakeup` 去禁用键盘唤醒

**Reply by MuelNova** [reply]
感谢分享！其实我在之后还是遇到不少问题，最后还是选择了一个 Windows11 + ArchWSL 的配置。可惜就是 win 颜值实在不如 hyprland 之类的配置完成的颜值，不过胜在稳定。

**Comment by terrancewong** [comment]
感谢！

我用 Debian testing, kernel 6.12 (6.12.16), 键盘，蓝牙都没问题。

还是有几个的问题

1. 据说 UMAF 可以解锁  USB4, 但是 我在 1.09 BIOS 下操作 遇到 “Submit fail”, 不知道有没有招？
2. 网卡到底是 2.5GbE 还是 GbE？我的卡在 1GB. 而且测速有点慢，iperf3 双向:发 900M，收1xxM
3. 4060 貌似有问题， 装了535 驱动， 但是 `clinfo` 看不到，`nvidia-smi` 也没有. 待会看看 dmesg

**Comment by SteveZMTstudios** [comment]


**Reply by SteveZMTstudios** [reply]
~~还有这个kernel cmdline怎么加呢~~

**Comment by terrancewong** [comment]
`gpiolib_acpi.ignore_interrupt=AMDI0030:00@6` 可以的
我试过 `gpiolib_acpi.ignore_interrupt=AMDI0030:00@6 gpiolib_acpi.ignore_interrupt=AMDI0030:00@3` 不太行
